rocky/rocky/views/ooi_view.py

```diff
@@ -150,7 +151,7 @@ def get_current_ooi(self) -> OOI | None:
         if self.observed_at.date() == now.date():
             return self.ooi
```

Comment:
Is this check still needed? the tree uses the self.observed_at property of the view anyway.

Reply:
I'm not sure. That might apply to the tree-based views, but I think this check is also used within the detail view